inference_segmentation.py

```diff
@@ -1,15 +1,16 @@
 import csv
-from math import sqrt
+import rasterio
+
```

Comment:
it should look like this:

```
from shutil import move
from pathlib import Path
from numbers import Number
from tempfile import mkstemp
from typing import Dict, Sequence, Union

from tqdm import tqdm
import csv
import rasterio
from omegaconf import DictConfig

from utils.aoiutils import aois_from_csv
from dataset.stacitem import SingleBandItemEO
from utils.logger import get_logger, set_tracker
from geo_inference.geo_inference import GeoInference
from utils.utils import get_device_ids, get_key_def, set_device
```

but I think that PEP8 refactoring can be done later in a separate PR.